docs/run/run-jobs-in-session.mdx

```diff
@@ -103,32 +107,32 @@ with Session(backend=backend):
 
 <span id="close"></span>
-## Close a session 
+## Close a session
 
-A session automatically closes when it exits the context manager. With qiskit-ibm-runtime 0.13 or later releases, when the session context manager is exited, the session is put into "In progress, not accepting new jobs" status. This means that the session finishes processing all running or queued jobs until the maximum timeout value is reached. After all jobs are completed, the session is immediately closed. This allows the scheduler to run the next job without waiting for the session interactive timeout, thereby reducing the average job queuing time. You cannot submit jobs to a closed session.
+A session automatically closes when it exits the context manager. With `qiskit-ibm-runtime` 0.13 or later releases, when the session context manager is exited, the session is put into "In progress, not accepting new jobs" status. This means that the session finishes processing all running or queued jobs until the maximum timeout value is reached. After all jobs are completed, the session is immediately canceled. This allows the scheduler to run the next job without waiting for the session interactive timeout, thereby reducing the average job queuing time. You cannot submit jobs to a closed session.
 
 ``` python
 with Session(service=service, backend=backend) as session:
     estimator = Estimator()
     job1 = estimator.run(...)
     job2 = estimator.run(...)
-    
-# The session is no longer accepting jobs but the submitted job will run to completion.    
+
+# The session is no longer accepting jobs but the submitted job will run to completion.
 result = job1.result()
 result2 = job2.result()
 ```
 
-If you are not using a context manager, it's good practice to manually close the session once all the necessary results have been retrieved. With qiskit-ibm-runtime 0.13 or later releases, when a session is closed with `session.close()`, it no longer accepts new jobs, but the already submitted jobs will still run until completion and their results can be retrieved. Prior to qiskit-ibm-runtime 0.13, when a session is closed with `session.close()`, any jobs that are already running continue to run, but any queued jobs remaining in the session are put into a failed state.
+If you are not using a context manager, manually close the session after all the necessary results have been retrieved. With `qiskit-ibm-runtime` 0.13 or later releases, when a session is closed with `session.close()`, it no longer accepts new jobs, but the already submitted jobs will still run until completion and their results can be retrieved. Prior to qiskit-ibm-runtime 0.13, when a session is closed with `session.close()`, any jobs that are already running continue to run, but any queued jobs remaining in the session are put into a failed state.
```

Comment:
```suggestion
If you are not using a context manager, you should manually close the session to avoid unwanted cost. With `qiskit-ibm-runtime` 0.13 or later releases, you can close a session as soon as you are done submitting jobs to it. When a session is closed with `session.close()`, it no longer accepts new jobs, but the already submitted jobs will still run until completion and their results can be retrieved. Prior to qiskit-ibm-runtime 0.13, when a session is closed with `session.close()`, any jobs that are already running continue to run, but any queued jobs remaining in the session are put into a failed state.
```